src/developer/web-api/tracker.md

```diff
@@ -1994,6 +1997,26 @@ F8yKM85NbxW,Zy2SEgA61ys,2019-08-21T11:25:38.022Z,2019-03-19T00:12:16.624Z,2019-0
 F8yKM85NbxW,Zy2SEgA61ys,2019-08-21T11:25:38.022Z,2019-03-19T00:12:16.624Z,2019-08-21T11:31:33.410Z,2019-03-19T00:12:16.624Z,DiszpKrYNg8,false,false,false,"POINT (-11.7896 8.2593)",8.2593,-11.7896,,,,2019-08-21T11:25:38.388Z,2019-08-21T11:25:38.388Z,aW66s2QSosT,"Last Name",Johnson,TEXT
 ```
 
+##### Tracked Entities Collection limits
+
+The collection endpoint limits results in three ways:
+- **System settings**: `KeyTrackedEntityMaxLimit` defines the maximum tracked entities in an API 
+response, protecting database and server resources. No limit applies when set to 0. Configure it via
+`/api/systemSettings` as described in the [documentation](settings-and-configuration.md?#webapi_system_settings).
+- **Program or tracked entity type**: it limits results when searching **outside
```

Comment:
I feel the heading and description is a out of sync. Maybe we should specify which "flag/parameter" of the Program and TET we are referring to here?